docker/test/stress/run.sh

```diff
@@ -233,4 +233,10 @@ rowNumberInAllBlocks()
 LIMIT 1" < /test_output/test_results.tsv > /test_output/check_status.tsv || echo "failure\tCannot parse test_results.tsv" > /test_output/check_status.tsv
 [ -s /test_output/check_status.tsv ] || echo -e "success\tNo errors found" > /test_output/check_status.tsv
 
+# But OOMs in stress test are allowed
+if rg 'OOM in dmesg|Signal 9' /test_output/check_status.tsv
+then
+    sed -i 's/failure/success/' /test_output/check_status.tsv
```

Comment:
Should we keep "failure" status for debug builds? Memory tracking and memory consumption in debug builds should not be much different from release builds, so OOM with debug build may indicate a real problem

Reply:
I've seen it frequently in debug builds. I'm not sure what the reason is - maybe the manual ASLR that we do.